### PR TITLE
Blog Improvements

### PR DESCRIPTION
**Pull request by BrandonMathis**
## What Changed
- Update Rails
- Install paranoia
- Lock Taperole at 1.2.8
- Fix tests to work with new version of rails
## Why
- Ellen "Accidentally" Deleted the entire General wiki article yesterday. This makes sure that you cannot permanently delete an article in the wiki
- It's always good to keep rails up to date

### CODE REVIEW

File: app/views/articles/show.html.haml [hunk 1]
```diff
@@ -51,7 +51,7 @@
             - toggle_archived_label = !@article.archived? ? 'Archive' : 'Unarchive'
             = link_to toggle_archived_label, toggle_archived_article_url( @article ), class: 'btn btn--b', method: :put
           %li.list-item
-            = link_to 'Delete', @article, class: 'btn btn--b', method: :delete, data: {confirm: "Are you sure?"}
+            = link_to 'Delete', @article, class: 'btn btn--b', method: :delete, data: {confirm: "YOU ARE ABOUT TO DELETE THIS ARTICLE FOREVER!!!! ARE YOU SURE?"}
```
Comment (mweslander) [inline app/views/articles/show.html.haml]: Lol. Really? +1 for clarity.

**Comment by mweslander** [comment]
Could use some ruby eyes. I'll make a note in slack.

File: db/structure.sql [hunk 2]
```diff
@@ -2,12 +2,16 @@
 -- PostgreSQL database dump
```
Comment (dkniffin) [inline db/structure.sql:2]: What is this file for? I've never seen this. Is this instead of a schema.rb or something?

**Comment by dkniffin** [comment]
LGTM. 👍

File: db/structure.sql (hunk 2 above)
Comment (joeyjoejoejr) [inline db/structure.sql:2]: Yeah.  You need it if you have non-standard database setup.

Comment (BrandonMathis) [inline db/structure.sql:2]: Yea, Orientation uses some special database foo that requires you to dump your schema like this.

**Pull request merged**